### DISCUSSION TITLE
Table row expand on click

### DISCUSSION THREAD

**Discussion by dev-federico** [filamentphp/filament#8673] [Help]
Hello,

is there a way to expand a table row by clicking on it?
I mean instead of opening the edit page when clicking on a row I would like to expand it.
Thank you in advance!

**Comment by leandrocfe** [comment]
Not sure about it, but you can have a look [collapsible-content](https://filamentphp.com/docs/3.x/tables/layout#collapsible-content) and [collapsible-groups](https://filamentphp.com/docs/3.x/tables/grouping#collapsible-groups)

**Reply by hrodrigues-belodigital** [reply]
I took a look to [https://filamentphp.com/docs/3.x/tables/layout#collapsible-content](collapsible-content) and I found it quite useful and I managed to do what I wanted. My goal was to have the parent show as a row and have the children show when the row it's not collapsed. What I am wondering now is if it is possible to show the column names.

**Comment by dev-federico** [comment]
Hi, thanks for the suggestion but this is not what I'm looking for.
I need to be able to expand the row by clicking on it, not by clicking on the default button on the right
